code/controllers/subsystem/timer.dm

```diff
@@ -291,18 +302,19 @@ SUBSYSTEM_DEF(timer)
 		// Insert the timer into the bucket, and perform necessary circular doubly-linked list operations
```

Comment:
Is it a circular doubly-linked list, or is it not?

Reply:
Thank you. Regular doubly-linked list. Corrected.